### PR TITLE
NXBT-2178: cleanup test tomcat after tests, to reduce target dir foot…

### PR DESCRIPTION
**Pull request by atchertchian**
…print

### CODE REVIEW

**Comment by kevinleturc** [comment]
👍

**Comment by jcarsique** [comment]
:-1: wait before merge

File: src/main/resources/nuxeo-ftest.xml [hunk 1]
```diff
@@ -1553,7 +1553,14 @@
           description="prepare all ressources for test">
   </target>
 
-  <target name="cleanup-environment" depends="_init,cleanup-db" />
+  <target name="cleanup-tomcat">
```
Comment (jcarsique) [inline src/main/resources/nuxeo-ftest.xml]: only if "is tomcat"
`  <target name="prepare-tomcat" depends="_init" if="is.tomcat">`

File: src/main/resources/nuxeo-ftest.xml [hunk 2]
```diff
@@ -1553,7 +1553,14 @@
...
+    <delete dir="${nuxeo.home}" failonerror="false" includeemptydirs="true">
```
Comment (jcarsique) [inline src/main/resources/nuxeo-ftest.xml]: if `env.NUXEO_HOME` is defined, then `nuxeo.home` is local (out of Maven target) and must not be deleted.

**Pull request merged**